vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -9,7 +9,7 @@
 import torch
 import torch.distributed as dist
 from torch.distributed import ProcessGroup
-from zmq import PUB, REP, REQ, SUB, SUBSCRIBE, Context  # type: ignore
+from zmq import SUB, SUBSCRIBE, XPUB, XPUB_VERBOSE, Context  # type: ignore
```

Comment:
Do we need `XSUB`, too?

Reply:
No, you can check https://netmq.readthedocs.io/en/latest/xpub-xsub/ .

`XPUB` connects to `SUB`.

`XSUB` is used to connect many `PUB`, which is not our usecase.